libsolidity/AST.h

```diff
@@ -1217,10 +1217,10 @@ class Identifier: public PrimaryExpression
 	}
 	Declaration const& getReferencedDeclaration() const;
 
-	/// Stores a set of possible declarations referenced by this identifier. Has to be resolved
```

Comment:
I think we can still say "set" - in the mathematical/colloquial and not data-structure sense.